pyensign/events.py

```diff
@@ -33,18 +30,10 @@ def __init__(self, data=None, mimetype=None, id="", meta={}):
         if not data:
             raise ValueError("no data provided")
 
-        if not mimetype:
+        if mimetype is None:
             raise ValueError("no mimetype provided")
```

Comment:
Should we have a default mimetype as the keyword argument rather than requiring the user to specify a valid one here?

Reply:
Yes! Think it makes sense to default to json?